[2025-04-24T15:31:20.483+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T15:31:20.491+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_lab.create_table manual__2025-04-24T15:31:19.134030+00:00 [queued]>
[2025-04-24T15:31:20.497+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_lab.create_table manual__2025-04-24T15:31:19.134030+00:00 [queued]>
[2025-04-24T15:31:20.497+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-24T15:31:20.505+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): create_table> on 2025-04-24 15:31:19.134030+00:00
[2025-04-24T15:31:20.509+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1191) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-24T15:31:20.511+0000] {standard_task_runner.py:72} INFO - Started process 1200 to run task
[2025-04-24T15:31:20.511+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '***_lab', 'create_table', 'manual__2025-04-24T15:31:19.134030+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpnp47uj_d']
[2025-04-24T15:31:20.514+0000] {standard_task_runner.py:105} INFO - Job 77: Subtask create_table
[2025-04-24T15:31:20.548+0000] {task_command.py:467} INFO - Running <TaskInstance: airflow_lab.create_table manual__2025-04-24T15:31:19.134030+00:00 [running]> on host 7ce975cba66d
[2025-04-24T15:31:20.594+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***_lab' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T15:31:19.134030+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T15:31:19.134030+00:00'
[2025-04-24T15:31:20.594+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-24T15:31:20.596+0000] {logging_mixin.py:190} INFO - Creating table: TB_SYMPTOMS_GGSHEET
[2025-04-24T15:31:20.602+0000] {base.py:84} INFO - Retrieving connection 'snowflake_default'
[2025-04-24T15:31:20.603+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.2, Python Version: 3.12.6, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-24T15:31:20.604+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-24T15:31:20.605+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-24T15:31:21.036+0000] {logging_mixin.py:190} INFO - Connected to Snowflake: <snowflake.connector.connection.SnowflakeConnection object at 0xffff88f3b260>
[2025-04-24T15:31:24.068+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-24T15:31:24.554+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-24T15:31:24.650+0000] {connection.py:788} INFO - closed
[2025-04-24T15:31:24.972+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-04-24T15:31:25.299+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-24T15:31:35.966+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-24T15:31:35.971+0000] {job.py:229} INFO - Heartbeat recovered after 15.52 seconds
[2025-04-24T15:31:35.972+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=***_lab, task_id=create_table, run_id=manual__2025-04-24T15:31:19.134030+00:00, execution_date=20250424T153119, start_date=20250424T153120, end_date=20250424T153135
[2025-04-24T15:31:36.031+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T15:31:36.069+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-24T15:31:36.074+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
